lib/collector/remote-method.js

```diff
@@ -16,7 +16,11 @@ const Sink = require('../util/stream-sink')
 const agents = require('./http-agents')
 const certificates = require('./ssl/certificates')
 const isValidLength = require('../util/byte-limit').isValidLength
-const NAMES = require('../metrics/names')
+const { SUPPORTABILITY, DATA_USAGE } = require('../metrics/names')
+
+function getMetricName(name) {
+  return `${SUPPORTABILITY.NODEJS}/Collector/${name}/${DATA_USAGE.SUFFIX}`
```

Comment:
you created a `DATA_USAGE.COLLECTOR` to take care of this prefix. I'd update it here

Reply:
`DATA_USAGE.COLLECTOR` is `${SUPPORTABILITY.NODEJS}/Collector/${DATA_USAGE.SUFFIX}`. Are you saying I should modify it to just `${SUPPORTABILITY.NODEJS}/Collector`?

Reply:
oh i see.  I'm suggesting creating yet another constant so you don't have the raw string of `Collector` here

Reply:
or you could pare the `DATA_USAGE.COLLECTOR` to exclude the suffix and just construct that below with 2 diff constants

Reply:
all valid approaches. i'll follow up when i can

Reply:
I've split it up into a prefix and a suffix.